src/core/settings.jl

```diff
@@ -1,7 +1,7 @@
 struct Settings
     horizon::Base.RefValue{Int}
     use_forecast_data::Bool
-    use_parameters::Bool
+    use_parameters::Base.RefValue{Bool}
```

Comment:
I think we should use `Ref{T}` instead of `Base.RefValue{T}` everywhere.

```suggestion
use_parameters::Ref{Bool}
```

If you use `Ref{Bool}`, you can pass `true` instead of `Ref(true)` to the constructor at the call site and it is stored as a `Ref` in the struct anyway.
If you use `Base.RefValue{Bool}`, you _have_ to use `Ref(true)` for the constructor at the call site. 

See screenshot for an example of this:

<img width="1430" alt="Screen Shot 2021-02-16 at 3 59 17 AM" src="https://user-images.githubusercontent.com/1813121/108054005-594b7280-700b-11eb-8e86-df1f10b6e8d6.png">

I think is more ergonomic to use `Ref{T}` everywhere.

Reply:
RefValue is used here to allow the field to be mutable. I tested the change and using ref doesn't allow to mutate the field in the same way. 

```julia 
julia> struct foo 
       val::Base.Ref{Bool}
       val_mut::Base.RefValue{Bool}
       end

julia> test = foo(true, Base.RefValue(true))
foo(Base.RefValue{Bool}(true), Base.RefValue{Bool}(true))

julia> test.val_mut[] = false
false

julia> test.val = false
ERROR: setfield! immutable struct of type foo cannot be changed
Stacktrace:
 [1] setproperty!(::foo, ::Symbol, ::Bool) at ./Base.jl:34
 [2] top-level scope at REPL[6]:1
 ```